packages/docusaurus-theme-classic/src/theme/SearchModal/icon.tsx

```diff
@@ -0,0 +1,479 @@
+      />
+      <path
+        fill="#ffb8b8"
+        d="M839.217 446.712l-9 23-45 42s-27 20-15 26 23-22 23-22l48-29 10-33z"
+        transform="translate(-200.148 -146.406)"
+      />
+      <path
+        fill="#ffb8b8"
+        d="M654.068 555.306L663.068 578.306 676.068 578.306 665.068 551.306 654.068 555.306z"
+      />
+      <path
+        fill="#ffb8b8"
+        d="M693.068 565.306L702.068 588.306 715.068 588.306 704.068 561.306 693.068 565.306z"
+      />
+      <path
+        fill="#2f2e41"
+        d="M892.217 715.712l17-4-22-89 7.304-68.48.196-.02s12.5-48.5 2.5-61.5-60-8-60-8-19 19-18 62 0 80 7 97 27 62 27 62l16-2-17-74-2.718-64.327c2.163 25.778 6.517 74.659 8.718 78.327 3 5 34 72 34 72z"
+        transform="translate(-200.148 -146.406)"
+      />
+      <path
+        fill="#ffb8b8"
+        d="M943.217 467.712l7 25 31 56s27 24 16 27-28-22-28-22l-35-49-9-36z"
+        transform="translate(-200.148 -146.406)"
+      />
+      <circle cx="707.068" cy="225.306" r="25" fill="#ffb8b8" />
+      <path
+        fill="#ffb8b8"
+        d="M894.217 384.712l-6 19-10 22 37-15s-2-16 0-20z"
+        transform="translate(-200.148 -146.406)"
+      />
+      <path
+        className={classes.svgFill}
+        d="M937.217 424.712l-23-19-26 11 1.018-16.226-27.74 8.86-4.278 1.366-6 31s-4 3-8 20a146.503 146.503 0 01-10 29s43-17 64 6c0 0 3-2 2-5s9-18 9-18 0 3 4-3 7-7 7-7z"
+        transform="translate(-200.148 -146.406)"
+      />
+      <path
+        className={classes.svgFill}
+        d="M863.217 413.712l-4.368-3.521s-3.632-1.48-6.632 3.52-18 36-18 36l18 9zM923.217 423.712l14 1s8 4 8 10 2 38 2 38l-22 7-9-28z"
+        transform="translate(-200.148 -146.406)"
+      />
+      <path
+        fill="#2f2e41"
+        d="M860.327 721.212l-17 7s-20 1-17 11 33 3 33 3 27 1 27-6-4-15-8-15z"
+        transform="translate(-200.148 -146.406)"
+      />
+      <path
+        fill="#2f2e41"
+        d="M898.327 730.212l-17 7s-20 1-17 11 33 3 33 3 27 1 27-6-4-15-8-15z"
+        transform="translate(-200.148 -146.406)"
+      />
+      <path
+        fill="#2f2e41"
+        d="M732.09 204.989L711.979 194.363 684.208 198.71 678.462 224.307 692.765 223.753 696.761 214.348 696.761 223.598 703.361 223.342 707.191 208.37 709.585 224.307 733.048 223.824 732.09 204.989z"
+      />
+      <path
+        fill="#3f3d56"
+        d="M797.728 229.953a135.02 135.02 0 107.655 199.402l165.619 139.748a12.442 12.442 0 1016.047-19.018L821.431 410.337a135.027 135.027 0 00-23.703-180.384zM787.052 396.88a101.158 101.158 0 11-12.077-142.548 101.158 101.158 0 0112.077 142.548z"
+        transform="translate(-200.148 -146.406)"
+      />
+      <path
+        d="M644.504 408.957a101.163 101.163 0 01-17.166-135.99q-2.901 2.923-5.61 6.12A101.158 101.158 0 10776.354 409.56q2.702-3.202 5.09-6.559a101.163 101.163 0 01-136.94 5.957z"
+        opacity="0.3"
+        transform="translate(-200.148 -146.406)"
+      />
+    </svg>
+  );
+}
+
+export default TrySearchIcon;
```

Comment:
quite large svg comps

I wonder if it would be more performant to keep these as regular SVGs so that the browser naturally lazy load them

Reply:
I wanted to add class to change SVG colors based on primary color